services/motion/builtin/move_request.go

```diff
@@ -267,6 +293,9 @@ func (ms *builtIn) newMoveOnGlobeRequest(
 	if straightlineDistance > maxTravelDistanceMM {
 		return nil, fmt.Errorf("cannot move more than %d kilometers", int(maxTravelDistanceMM*1e-6))
 	}
+	// NOTE: I don't understand what a referenceframe.Limit & that the mins & maxes describe.
```

Comment:
A frame has a method, `Transform`, which accepts an arbitrary number of floating point values to turn into some pose, wrapped in the `referenceframe.Input` type. The Limits of that frame specify 1) how many floats may be passed in, and 2) what the range of those values may be.